main.js

```diff
@@ -1,17 +1,10 @@
-import search from './src/js/search';
-import preventScroll from './src/js/prevent-scroll';
-import toggle from './src/js/toggle';
-import selected from './src/js/selected';
-import editionSwitcher from './src/js/edition-switcher';
+import oHeader from './src/js/Header';
 
-const init = flags => {
-	search.init(flags);
-	preventScroll.init();
-	toggle.init(flags);
-	selected.init(flags);
-	editionSwitcher.init(document.querySelector('.edition-switcher'), flags);
+const constructAll = () => {
+	oHeader.init();
+	document.removeEventListener('o.DOMContentLoaded', constructAll);
 };
 
-export default {
-	init
-};
+document.addEventListener('o.DOMContentLoaded', constructAll);
+
+export default oHeader;
```

Comment:
We're already using it in some modules for tests, and Next is heavily using it. The spec is stabilising as well and it's going to be implemented in browsers